.travis.yml

```diff
@@ -21,7 +21,7 @@ matrix:
 branches:
   only:
     - master
-    - "1.9"
```

Comment:
Since `1.10` is still `rc`, IMHO we should support `1.9`. POV @lserwatka @andrerom?

Reply:
Why? It it should point to 1.10 branch.

Reply:
This is 1.10 branch, so it should be 1.10, otherwise 1.10 will never be built.


1.9 will still be built ;)

Reply:
> This is 1.10 branch (...)
> 1.9 will still be built ;)

Ahm... yes, good point :)